lib/elixir/lib/partition_supervisor.ex

```diff
@@ -112,8 +113,8 @@ defmodule PartitionSupervisor do
 
   ## Options
 
-    * `:name` - an atom representing the name of the partition supervisor
-      (see `t:name/0`).
+    * `:name` - an atom, global, or via tuple representing the name of the
+      partition supervisor (see `t:name/0`).
```

Comment:
I actually don't think it can be global because we lookup on local tables. So we should say something like

```suggestion
    * `:name` - an atom or via tuple representing the name of the
      partition supervisor (see `t:name/0`). The name must always
      resolve to a process located in the current node.
```